.github/workflows/meson.yml

```diff
@@ -36,26 +71,42 @@ jobs:
           # Set up cc/c++ executables
           cc: 1 # optional, default is 1
 
+      - name: Setup Meson
+        env:
+          CC: "gcc"
+          CXX: "g++"
+        run: |
+          meson setup --buildtype=${{inputs.build_type}} -Ddebug_type=${{inputs.debug_level}} build
+
+      - name: Configure for AppImage
+        if: ${{inputs.upload_artefacts}}
+        env:
+          CC: "gcc"
+          CXX: "g++"
+        run: meson configure -Dinstall_data=false -Dinstall_runner=false -Dfmod_dir=/usr/lib/ --prefix=/usr/ build
```

Comment:
I'm curious, what has changed that makes this configure step here necessary? Sorry if this might be obvious, I know quite little about our meson stuff haha

Reply:
This is a @HeliumAnt question, but as far as I can explain it:

That code was there before, I've just conditionalised it only to run when we're packaging the Linux build as an AppImage to upload for later use. 

Afaik, it's required that we configure those values before we [install the game into a new AppDir here](https://github.com/cortex-command-community/Cortex-Command-Community-Project-Source/blob/e3a7c448612fda3de9d2f6f3af2d140b68ef1ccc/.github/workflows/meson.yml#L99) so that meson packages it correctly